src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets

```diff
@@ -54,23 +54,19 @@ Copyright (c) .NET Foundation. All rights reserved.
       $(GetCurrentProjectStaticWebAssetsDependsOn)
     </GetCurrentProjectStaticWebAssetsDependsOn>
 
-    <AssignTargetPathsDependsOn>
+    <GetCopyToOutputDirectoryItemsDependsOn>
+      $(GetCopyToOutputDirectoryItemsDependsOn);
       GenerateStaticWebAssetsManifest;
-      $(AssignTargetPathsDependsOn)
-    </AssignTargetPathsDependsOn>
+    </GetCopyToOutputDirectoryItemsDependsOn>
 
     <ResolveStaticWebAssetsInputsDependsOn>
       ResolveCurrentProjectStaticWebAssetsInputs;
-      $(ResolveStaticWebAssetsInputsDependsOn)
-    </ResolveStaticWebAssetsInputsDependsOn>
-
-    <ResolveStaticWebAssetsInputsDependsOn Condition="$(NoBuild) != 'true'">
       ResolveReferencedProjectsStaticWebAssets;
       $(ResolveStaticWebAssetsInputsDependsOn)
     </ResolveStaticWebAssetsInputsDependsOn>
 
     <ResolveReferencedProjectsStaticWebAssetsDependsOn>
-      ResolveReferences;
+      PrepareProjectReferences;
```

Comment:
What was the reason for this change?

Reply:
Fixes a trivial `pack —no-build` bug that was hidden under the `NoBuild` check.

Reply:
There’s nothing that depends on NoBuild now,  which is much better.